client/app/visualizations/word-cloud/Editor.jsx

```diff
@@ -0,0 +1,35 @@
+import { map } from 'lodash';
+import React from 'react';
+import Select from 'antd/lib/select';
+import { EditorPropTypes } from '@/visualizations';
+
+const { Option } = Select;
+
+export default function Editor({ options, data, onOptionsChange }) {
+  const onColumnChanged = (column) => {
+    const newOptions = { ...options, column };
+    onOptionsChange(newOptions);
+  };
+
+  return (
+    <div className="form-horizontal">
+      <div className="form-group d-flex align-items-center">
+        <label className="col-lg-6" htmlFor="word-cloud-column">Word Cloud Column Name</label>
+        <div className="col-lg-6">
+          <Select
+            id="word-cloud-column"
+            className="w-100"
+            value={options.column}
+            onChange={onColumnChanged}
```

Comment:
Add `dropdownMatchSelectWidth` cause the default value is '' which makes the dropdown squished.

Reply:
@ranbena Can you explain what do you mean? Seems it's okay:

![image](https://user-images.githubusercontent.com/12139186/53873397-f96b8500-4008-11e9-87a1-d85cbf0148ca.png)

Reply:
1. Existing query
2. Click New Visualization
3. Select Word Cloud

![screen shot 2019-03-06 at 12 13 14](https://user-images.githubusercontent.com/486954/53873630-6ed75580-4009-11e9-8656-d8b0afe7bbdc.png)

Reply:
And then
![screen shot 2019-03-06 at 12 15 42](https://user-images.githubusercontent.com/486954/53873715-97f7e600-4009-11e9-8b76-d997b0377a53.png)

Reply:
@ranbena Did everything as you described, but...

![image](https://user-images.githubusercontent.com/12139186/53875069-70564d00-400c-11e9-9adc-4b8772ab671c.png)

If you can investigate it - I'll appreciate your help (really cannot reproduce it).

Reply:
There is `w-100` class - entire `<Select>` should fit it's container. Can you check why it doesn't work for you?

Reply:
It has to do with the wrapping `className="col-lg-6"` which has no width defined when screen < 1200px.

Reply:
Got it, will fix

Reply:
Fixed